commons/acceptance_tests/initializedb.py

```diff
@@ -1,60 +1,16 @@
-import os
-import sys
-from typing import List
+# -*- coding: utf-8 -*-
 
 import transaction
-from logging.config import fileConfig
 
-from pyramid.paster import get_appsettings
-
-from pyramid.scripts.common import parse_vars
 from sqlalchemy.engine import Connection
 from sqlalchemy.orm import Session
 
 from c2cgeoportal_commons.models import (
-    Base, get_engine, get_session_factory, get_tm_session, generate_mappers,
+    Base, get_session_factory, get_tm_session,
 )
 from c2cgeoportal_commons.config import config
 
 
-def usage(argv: List[str]) -> None:
-    cmd = os.path.basename(argv[0])
-    print(
-        'usage: %s <config_uri> [var=value]\n'
-        '(example: "%s development.ini")' % (cmd, cmd)
-    )
-    sys.exit(1)
-
-
-def main(argv: List[str]=sys.argv) -> None:
-    if len(argv) < 2:
-        usage(argv)
-    config_uri = argv[1]
-    options = parse_vars(argv[2:])
-
-    fileConfig(config_uri, defaults=dict(os.environ))
-    options.update(os.environ)
-    settings = get_appsettings(config_uri, options=options)
-    generate_mappers()
-
-    engine = get_engine(settings)
-
-    with engine.begin() as connection:
-        init_db(
-            connection,
-            force='--force' in options,
-            test='--test' in options
-        )
```

Comment:
Is it not possible to keep an entry point initialize_db script in commons package ?

Reply:
I keep it on commons but not as an entrypoint...